rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/RangeTest.java

```diff
@@ -39,9 +39,9 @@ void parenthesized() {
         rewriteRun(
           groovy(
             """
-              ( 8..19 ).each { majorVersion ->
+              (((( 8..19 ))).each { majorVersion ->
                 if (majorVersion == 9) return
-              }
+              })
```

Comment:
I'm surprised we're changing the original here as well; would it make sense to add a second `groovy` block within the same `rewriteRun` instead?